README.md

```diff
@@ -37,7 +37,7 @@ I would like to:
 
 - join a small group to receive mentoring from experienced instructors OR
 - mentor new instructors
-	- Sign up for the mentoring program by contacting Kari L. Jordan (kariljordan@carpentries.org)
+	- Join a mentoring group by contacting Kari L. Jordan (mailto:kariljordan@carpentries.org)
```

Comment:
To make it consistent with the link that you made in program-outline.md in #22, I would have a 'proper' Markdown link i.e. `[Kari L. Jordan](mailto:kariljordan@carpentries.org)` here too.